Automattic/jetpack
# Initialize Backup UI from package

Loads the Backup UI from the Backup package

#### Changes proposed in this Pull Request:

* Adds Jetpack Backup UI to Jetpack

#### Jetpack product discussion


[need from leo]

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* With a Jetpack site with a plan that includes Backup, confirm that only with Jetpack, the Backup UI is accessible.
* Check Jetpack without a plan.
